oio/cli/object/object.py

```diff
@@ -45,6 +49,10 @@ def patch_parser(self, parser):
                   "'flat_*' namespace parameters (<container> is ignored)."),
             action="store_true",
         )
+        parser.add_argument(
+            '--flat-bitlength',
+            help=("Override NS flatns for --auto"),
+        )
```

Comment:
I suggest naming the option `--flat-bits`, as in the local configuration file (`ns.flat_bits`), and define it as "Number of bits for flat-NS computation".